mobile/geth.go

```diff
@@ -185,7 +218,56 @@ func NewNode(datadir string, config *NodeConfig) (stack *Node, _ error) {
 			return nil, fmt.Errorf("whisper init: %v", err)
 		}
 	}
-	return &Node{rawStack}, nil
+	if config.PssEnabled && ks != nil {
+		log.Debug("pss enabled")
+		bzzServiceFunc := func(ctx *node.ServiceContext) (node.Service, error) {
+			log.Debug("keystore", "ks", ks)
+			var a accounts.Account
+			var err error
+			if common.IsHexAddress(config.PssAccount) {
```

Comment:
This is not a safe way to access accounts. It should be done explicitly using address.